docs/eog/case-studies-cw20-agenda.rst

```diff
@@ -0,0 +1,76 @@
+.. _CW20-Agenda: 
+
+CW20 Agenda
+============
+
+For in-person CWs, the days are long to maximise the opportunities for networking, collaboration, discussions, and solving problems across the research software landscape. 
+The agenda that we had spent months organising was packed full of talks, group sessions, social activities and mini-workshops - we originally had 18 mini-workshops scheduled to take place across five sessions.
+
+If we had organised CW20 as an online event from the beginning, there are many things we would have done differently - but since we found ourselves in the position of having to transition from an in-person event to an online one, we still wanted to incorporate as much of the original programme as possible. 
```

Comment:
```suggestion
If we had organised CW20 as an online event from the beginning, there are many things we would have done differently - but since we found ourselves in the position of having to transition from an in-person event to an online one at short notice, we still wanted to incorporate as much of the original programme as possible. 
```